Composer/packages/client/src/pages/skills/skill-list.tsx

```diff
@@ -84,7 +86,10 @@ const SkillList: React.FC<ISkillListProps> = props => {
   const { actions } = useContext(StoreContext);
 
   const { skills, projectId } = props;
+
   const [editIndex, setEditIndex] = useState<number | undefined>(undefined);
+  const [isModalOpen, setModalOpen] = useState<boolean>(false);
+  const [modalOpenIndex, setModalOpenIndex] = useState<number>(0);
```

Comment:
This could be combined into a single state value since they always change together.

```ts
const onViewManifest = (skill) => {
  setSelectedSkill(skill)
}

const onHideManifest = () => {
  setSelectedSkill(null)
}

isOpen={!!selectedSkill}
```